src/main/java/ua/danit/final_project/services/WorkCommentService.java

```diff
@@ -0,0 +1,24 @@
+package ua.danit.final_project.services;
+
+import ua.danit.final_project.entities.ShiftComment;
+import ua.danit.final_project.entities.WorkShift;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.entities.WorkShift'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=85&issues=AWWG_p8MX2sY87p04Y8C&open=AWWG_p8MX2sY87p04Y8C)

[ISSUE_KEY]: AWWG_p8MX2sY87p04Y8C